app/controllers/CommendSectionController.scala

```diff
@@ -0,0 +1,51 @@
+package controllers
+
+import javax.inject.Inject
+import models.Commends
+import models.JsonFormats._
+import play.api.mvc._
+import play.modules.reactivemongo.{MongoController, ReactiveMongoApi, ReactiveMongoComponents}
+import reactivemongo.play.json.collection.JSONCollection
+import services.ReleasedServices
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class CommendSectionController @Inject()(
+                                          components: ControllerComponents,
+                                          val reactiveMongoApi: ReactiveMongoApi,
+                                          val releasedServices: ReleasedServices
+                                        ) extends AbstractController(components)
+  with MongoController with ReactiveMongoComponents with play.api.i18n.I18nSupport {
+  implicit def ec: ExecutionContext = components.executionContext
+
+  def collection: Future[JSONCollection] = reactiveMongoApi.database.map(_.collection[JSONCollection]("comment"))
+
+
+  def getCommends = Action { implicit request: Request[AnyContent] =>
+    Ok(views.html.commends(Commends.createCommentForm))
+  }
+
+  def serviceSubmitComment(comment: Commends) = {
+    collection.flatMap(_.insert.one(comment))
+  }
+
+  def submitCommend = Action.async { implicit request: Request[AnyContent] =>
+    Commends.createCommentForm.bindFromRequest.fold({ formWithErrors =>
+      println(formWithErrors)
+      Future.successful(BadRequest(views.html.commends(formWithErrors)))
+    }, { Commends =>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Field name 'Commends' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5276325)